tests/js/searchtools.js

```diff
@@ -27,6 +27,33 @@ describe('Basic html theme search', function() {
       expect(Search.performTermsSearch(searchterms, excluded, terms, titleterms)).toEqual(hits);
     });
 
+    it('should be able to search for multiple terms', function() {
+      index = {
+        alltitles: {
```

Comment:
That's for another PR but the formatting style differs in this file so we should just make it the same (like, there are spaces after colons sometimes and sometimes not) and sometimes there are semicolons and sometimes not.

Reply:
IMO we should run JavaScript through [prettier](https://prettier.io/). Definitely out of scope here though.